### PR TITLE
chore: add info about releases

### PR DESCRIPTION
**Pull request by MatousJobanek**
add info about releases

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -85,6 +85,24 @@ That Makefile target takes care of several steps that can be executed separately
...
+== Releasing operator
+
+The releases of the operator are automatically managed via Tekton pipeline running under `codeready-toolchain` user account in CRT cluster.
+
+The definition of the CD pipeline can be found here https://github.com/codeready-toolchain/toolchain-cicd.
+The pipeline triggers Makefile targets located in this repository.
+
+=== Broken release
+
+If there is any broken release that cannot be built & pushed through the pipeline and has to be done/fixed manually then do the following steps:
```
Comment (rajivnathan) [inline README.adoc:97]: Same question as in the member-operator PR : https://github.com/codeready-toolchain/member-operator/pull/230#discussion_r543424877

**Review by rajivnathan — APPROVED** [review]
Nice, thanks for adding this! Have one question/suggestion

**Review by alexeykazakov — APPROVED** [review]

File: README.adoc (hunk 1 above)
Comment (MatousJobanek) [reply README.adoc]: done in ea23958

**Pull request merged**